spec/acceptance/basic_webhook_spec.rb

```diff
@@ -58,5 +58,17 @@ class {'r10k::webhook':
         expect(r.exit_code).to eq(0)
       end
     end
+
+    it 'should successfully lock when hammered with multiple requests' do
```

Comment:
These aren't really good validations. Having four separate `it` blocks implies that it's testing four different things, and it's not. It's testing one.

Not using the word "should" is fine, but all the other tests do, so that becomes inconsistent.

Not allowing a newline to delineate different blocks is just asinine, especially when theres a cascade of `end`s, like below. Which one ends which block?

```ruby
          end
        end
      end
    end
  end
end
```

Its style guide even praises using newlines. https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods but has nothing for or against newlines around blocks.

The regex rule linked to doesn't actually say to always use `%r`, it says "Use %r only for regular expressions matching *at least one* '/' character." This regex does not, it simply matches the word "success". I'd change it anyways to make the rule happy, but then this test would be inconsistent with the rest.

I fixed the `times.map` (wasn't really needed anyway), took out the readability whitespace, and added a disable comment for the multiple expectations. If you want the other rules to pass, you ought to make a single commit to change all of them at once for consistency.